src/actions/monteCarloActions.test.js

```diff
@@ -0,0 +1,20 @@
+import { ADD_1_DART, add1Dart, ADD_10_DARTS, add10Darts, ADD_100_DARTS, add100Darts, ADD_1000_DARTS, add1000Darts } from './monteCarloActions';
```

Comment:
importing from monteCarloActions but the file is montecarloActions, which may be causing the test to fail.